rsyslog.yaml

```diff
@@ -63,6 +159,66 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - range: plugins
+    name: rsyslog-${{range.key}}
+    description: rsyslog plugin for ${{range.key}}
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/lib/rsyslog/
+          PLUGINS="$(echo "${{range.value}}" | tr ',' ' ')"
+          for plugin in $PLUGINS; do
+            mv ${{targets.destdir}}/usr/lib/rsyslog/${plugin}.so ${{targets.contextdir}}/usr/lib/rsyslog/
+          done
+    test:
+      pipeline:
+        - runs: |
+            PLUGINS="$(echo "${{range.value}}" | tr ',' ' ')"
+            for plugin in $PLUGINS; do
+              stat /usr/lib/rsyslog/${plugin}.so
+            done
+
```

Comment:
We can consider doing a split/doc and split/dev here well. 💭

Reply:
`split/dev` might be a breaking change, so i'd keep that for another discussion tbh.